keyboards/owl8/config.h

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2021 yfuku
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x5946 // YF
+#define PRODUCT_ID 0x0008
+#define DEVICE_VER 0x0001
+#define MANUFACTURER yfuku
+#define PRODUCT      owl8
+
+/* key matrix size */
+#define MATRIX_ROWS 1
+#define MATRIX_COLS 16
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { NO_PIN }
+#define MATRIX_COL_PINS { F4, F7, B3, B6, F5, F6, B1, B2, D4, C6, D7, E6 }
```

Comment:
This needs to be swapped to DIRECT_PINS
<https://docs.qmk.fm/#/hardware_avr?id=direct-pin-matrix>

Reply:
@zvecr 
Thanks for the review.
Fixed it to use DIRECT_PINS.